src/server/utils/sendSentryEvent.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import getRethink from 'server/database/rethinkDriver';
+import {getUserId} from 'server/utils/authorization';
+import Raven from 'raven';
+
+type DataloaderData = {
+  key: string,
+  keys: Array<string>,
+  indexedResults: Map<string, any>
+}
+
+type GraphQLData = {
+  query: string,
+  variables?: { [name: string]: any },
+  firstError?: any
+};
+
+type MembershipData = {
+  teamId: string
+}
+
+type Data =
+  | DataloaderData
+  | GraphQLData
+  | MembershipData
+
+type Breadcrumb = {
+  message: string,
+  category: string,
+  data: Data
+};
+
+type AuthToken = {
+  sub: string
+}
+
+const sendSegmentEvent = async (authToken?: AuthToken, breadcrumb?: Breadcrumb) => {
```

Comment:
I don't think this should be called sendSegmentEvent...

Reply:
ah good catch. segment sentry slack stripe it's all the same, right?